insights/client/utilities.py

```diff
@@ -445,3 +447,41 @@ def os_release_info():
         except Exception as e:
             logger.warning("Failed to detect OS version: %s", e)
     return (os_family, os_release)
+
+
+def largest_spec_in_archive(archive_file):
+    logger.info("Checking for large files...")
+    tar_file = tarfile.open(archive_file, 'r')
+    largest_fsize = 0
+    # get the name of the archive
+    for elems in archive_file.split("/"):
+        if "tar" in elems:
+            name = elems.split(".", 1)
```

Comment:
Is `archive_file` an absolute path to the archive?

If yes, it would better to use `os.path` instead:

`name = os.path.basename(archive_file).split(".")[0]`

Reply:
Yes, it is

Reply:
Sorry, @ahitacat. I've misled you about this. Actually the archive name is always have this format `insights-$FQDN-$TIMESTAMP.tar.gz` and the FQDN contains dots :/ 

It should be:

`name = os.path.basename(archive_file).split('.tar.gz')[0]`